x-pack/elastic-agent/pkg/agent/transpiler/ast_test.go

```diff
@@ -375,6 +375,184 @@ func TestAST(t *testing.T) {
 	})
 }
 
+func TestInsert(t *testing.T) {
+	testcases := map[string]struct {
+		hashmap  map[string]interface{}
+		selector Selector
+		node     Node
+		expected *AST
+	}{
+		// "insert root": {
```

Comment:
Why is this commented out?

Reply:
good catch :D i commented out while debugging